ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/geospatial/connected/relations/TileFragmentRelationsAttributer.java

```diff
@@ -14,7 +17,7 @@ public TileFragmentRelationsAttributer(FragmentRepository fragmentRepository) {
 	}
 
 	public void addRelationsFromRootToBottom(Fragment rootFragment, Fragment tileFragments) {
-		TreeRelation relationToParentFragment = relationsAttributer.getRelationToParentFragment(tileFragments);
+		TreeRelation relationToParentFragment = tileFragments.getValueOfKey(FRAGMENT_KEY_TILE).orElse("").equals(DEFAULT_BUCKET_STRING) ? relationsAttributer.getDefaultRelation(tileFragments) : relationsAttributer.getRelationToParentFragment(tileFragments);
```

Comment:
Can you rewrite this? I find it very hard to read. The line is way too long to fit on my screen and contains too much logic. To understand what was going on here, I had to split it in parts.

Splitting this in two would probably help. First define the boolean and then create the relation based on the boolean